src/main/java/edu/harvard/iq/dataverse/authorization/providers/oauth2/oidc/OIDCAuthProvider.java

```diff
@@ -0,0 +1,263 @@
+package edu.harvard.iq.dataverse.authorization.providers.oauth2.oidc;
+
+import com.github.scribejava.core.builder.api.DefaultApi20;
+import com.nimbusds.oauth2.sdk.*;
```

Comment:
In general, we try not to use star imports. I regret to say that we don't mention this at http://guides.dataverse.org/en/4.18.1/developers/coding-style.html 😄

Reply:
I'm sorry, IntelliJ IDEA does this by default.
Should I mention checkstyle again to catch this sort of stuff?
We could create a Jenkins job for it...

Reply:
Oh and should I change this in a new PR as this one is already merged?